cfme/tests/distributed/test_appliance_replication.py

```diff
@@ -118,6 +118,11 @@ def configure_db_replication(db_address):
     """
     conf.set_replication_worker_host(db_address)
     flash.assert_message_contain("Configuration settings saved for CFME Server")
+    try:
+        sel.force_navigate("cfg_settings_currentserver_server")
+    except Exception:
```

Comment:
Should this only be handling [](https://selenium-python.readthedocs.org/en/latest/api.html#selenium.common.exceptions.UnexpectedAlertPresentException)`UnexpectedAlertPresentException`?